Woof/Woof/Model/Sitter/Sitter+Dummy.swift

```diff
@@ -20,5 +20,20 @@ extension Sitter {
             rating: 4.8,
             pricePerHour: 40.25
         )
+
+        /// The instance of dummy sitter list to use in preview of views.
+        static let sitters = [
+            // swiftlint:disable line_length
+            Sitter(id: UUID(), name: "Jane", surname: "Smith", phone: "0987654321", avatarUrl: Bundle.main.url(forResource: "EmilyDoeAvatar", withExtension: "jpg"), bio: "Certified dog trainer. Enjoys playing fetch and teaching new tricks.", rating: 4.8, pricePerHour: 20.0),
```

Comment:
nit: You have the URL which points nowhere in fact.
Should we use the URL of the existing local resource instead to show some pre-defined image/images?

Otherwise how you're going to test the images?

Reply:
Added more source images, fixed with https://github.com/ios-course/ironfoudation-team-project/pull/48/commits/40c23a6da6f5db98503299f296ce96ec0b830a09